modules/installation-creating-aws-vpc-localzone.adoc

```diff
@@ -6,7 +6,7 @@
 [id="installation-creating-aws-vpc-localzone_{context}"]
 = Creating a VPC that uses AWS Local Zones
```

Comment:
VPC does not use Local Zones. VPC should exist to create Local Zone subnet.

I would suggest follow the doc standard: [`Creating a VPC in AWS`](https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-user-infra.html#installation-creating-aws-vpc_installing-aws-user-infra)

```suggestion
= Creating a VPC in AWS
```

Reply:
:heavy_check_mark: